spectre-meltdown-checker.sh

```diff
@@ -662,6 +662,10 @@ check_variant2()
 			dd if=/dev/cpu/0/msr of=/dev/null bs=8 count=1 skip=9 2>/dev/null
 			if [ $? -eq 0 ]; then
 				pstatus green YES
+			elif grep -qi spec_ctrl /proc/cpuinfo; then
```

Comment:
I would prefer `-qw` here, as there is also `spec_ctrl_ibrs`.
btw I'm not yet sure whether `spec_ctrl_ibrs` means IBRS is compiled-in *and* ((the microcode is also there, regardless of the fact that IBRS is enabled or not) *or* (IBRS completely enabled))